pkg/containerutil/container_network_manager.go

```diff
@@ -401,7 +409,6 @@ func (m *hostNetworkManager) SetupNetworking(ctx context.Context, containerID st
 
 	// Prep the meta
 	hsMeta := hostsstore.Meta{
-		Namespace:  container.Labels[labels.Namespace],
```

Comment:
This property is useless (and was before IIRC).
The meta is being saved under a namespaced location in the store already.